src/program.js

```diff
@@ -276,6 +276,11 @@ Example: $0 --help run.
       requiresArg: true,
       type: 'array',
     },
+    'no-input': {
+      alias: 'no-interactive',
+      describe: 'Disable the interactive behaviour',
```

Comment:
Let's make this more specific to what it will do. How about this:

Disable all features that require standard input